server/risk_scores/risk_assessment.py

```diff
@@ -72,28 +72,33 @@ def get_incident_similarity(prev_incident: submit_schema.Form, current_incident:
     return coefficient * similarity
 
 
-def get_incident_recency(prev_incident: submit_schema.Form, current_incident: submit_schema.Form, timeframe):
+def get_incident_recency(prev_incident: submit_schema.Form,
```

Comment:
These autoformatting changes might result in merge conflicts with some incoming PRs.